core/basic-authorship/src/basic_authorship.rs

```diff
@@ -174,12 +179,16 @@ impl<Block, C, A> consensus_common::Proposer<<C as AuthoringApi>::Block> for Pro
 	type Create = Result<<C as AuthoringApi>::Block, error::Error>;
 	type Error = error::Error;
 
-	fn propose(&self, inherent_data: InherentData, max_duration: time::Duration)
-		-> Result<<C as AuthoringApi>::Block, error::Error>
+	fn propose(
+		&self,
+		inherent_data: InherentData,
+		max_duration: time::Duration,
+		inherent_digests: DigestFor<Block>,
```

Comment:
nit: can we swap the order of `max_duration` with `inherent_digests`? (I don't have a strong opinion on this so you can keep it as is if you think that's better).

Reply:
Done.